Services/source/housekeeping/housekeeping_service.c

```diff
@@ -225,13 +235,20 @@ Result mock_everyone(All_systems_housekeeping* all_hk_data) {
   all_hk_data->adcs_hk.Rate_Sensor_Temp_Y = tempFloat;
   all_hk_data->adcs_hk.Rate_Sensor_Temp_Z = tempFloat;
 
-  //Athena
+  // Athena
   uint8_t i;
-  for (i = 0; i < 6; i++) {
+  for (i = 0; i < ATHENA_TEMP_ARRAY_SIZE; i++) {
```

Comment:
added this macro. Clearly the clang-format didn't work terribly well. I'll try another commit tomorrow.

Reply:
Done